src/main.rs

```diff
@@ -1,26 +1,40 @@
 use crossterm::cursor::{Hide, Show};
-use crossterm::event::{Event, KeyCode};
 use crossterm::terminal::{EnterAlternateScreen, LeaveAlternateScreen};
-use crossterm::{event, terminal, ExecutableCommand};
-use invaders::frame::{new_frame, Drawable};
+use crossterm::{terminal, ExecutableCommand};
+use invaders::frame::{new_frame, Drawable, Frame};
 use invaders::invaders::Invaders;
+use invaders::menu::Menu;
 use invaders::player::Player;
 use invaders::score::Score;
+use invaders::Scenes;
 use invaders::{frame, render};
 use rusty_audio::Audio;
 use std::error::Error;
 use std::sync::mpsc;
+use std::sync::mpsc::Receiver;
 use std::time::{Duration, Instant};
 use std::{io, thread};
```

Comment:
I should have cleaned up these `use` statements long ago. Would you like to do that for me? This is optional, I will accept the pull request without this cleanup if you do not wish to do it.

If you do want to try doing the cleanup, here are some examples on how to combine `use` statements:

```rust
// These two lines...
use std::sync::mpsc;
use std::sync::mpsc::Receiver;

// would become this (`self` brings `mpsc` into scope)
use std::sync::mpsc::{self, Receiver};
```

Another example: 
```rust
// These lines...
use crossterm::cursor::{Hide, Show};
use crossterm::terminal::{EnterAlternateScreen, LeaveAlternateScreen};
use crossterm::{terminal, ExecutableCommand};

// would become
use crossterm::{
    cursor::{Hide, Show},
    terminal::{self, EnterAlternateScreen, LeaveAlternateScreen},
    ExecutableCommand,
};
```